Signet

BennyTheDev opened the pull request:
+ tx speed up thanks to supertestnet's hints
+ new design based on bootstrap (dark mode next)
+ added brc-20 minting and deployment

Commit:
# tx speed up thanks to supertestnet's hints
# new design based on bootstrap (dark mode next)
# added brc-20 minting and deployment

Commit:
Merge branch 'main' into main

supertestnet reviewed (changes requested):
please revert the css changes, I don't like this bootstrap stuff, it adds too much code to the page and will negatively impact the loading time

Commit:
# tx speed up thanks to supertestnet's hints
# new design based on bootstrap (dark mode next)
# added brc-20 minting and deployment

Commit:
Merge branch 'main' of https://github.com/BennyTheDev/inscriptions-online

Commit:
+ fixed broken isUsedDomain function

Commit:
+ fixed broken isUsedDomain function
